RELEASE-NOTES.txt

```diff
@@ -10,6 +10,7 @@
 * [Android] Add alignment options for heading block
 * Fix Quote block so it visually reflects selected alignment
 * Fix bug where buttons in page templates were not rendering correctly on web
+* New block: Pullquote
```

Comment:
Hey @ceyhun =) I think this should go in `1.28.0` because I believe this PR didn't make it for the code freeze of `1.27.0`

Reply:
That's right @geriux, thanks for pointing that out. Fixed it here: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2217